python/private/test/render_pkg_aliases_test.bzl

```diff
@@ -34,18 +41,19 @@ def render_pkg_aliases_test(name, **kwargs):
 
     outs = {}
     for file, contents in aliases.items():
-        want = name + "/" + file + ".want"
-        got = want.replace("/", "_").rstrip(".want")
+        want = name + "__" + file.replace("/", "__") + ".want"
+        got = want.rstrip(".want") + ".got"
         outs[got] = want
+
         write_file(
             name = got,
             out = got + "_",
             content = contents.splitlines(),
         )
 
         diff_test(
```

Comment:
@chrislovecnm, I am having a hard time with Windows tests and I am thinking that I should potentially mark them as incompatible with windows, but I am not sure it is the best way.

Reply:
@aignas I can take a look